nodejs/patches/0001-patch-no-execinfo.patch

```diff
@@ -0,0 +1,59 @@
+--- deps/v8/src/base/logging.cc.orig	2015-09-30 22:07:22.927437075 +0100
++++ deps/v8/src/base/logging.cc	2015-10-05 09:16:42.779003324 +0100
+@@ -7,7 +7,6 @@
+ #if V8_LIBC_GLIBC || V8_OS_BSD
+ #include <cxxabi.h>
+ #include <dlfcn.h>
+-#include <execinfo.h>
+ #elif V8_OS_QNX
+ #include <backtrace.h>
+ #endif  // V8_LIBC_GLIBC || V8_OS_BSD
+@@ -52,48 +51,6 @@
+ 
+ // Attempts to dump a backtrace (if supported).
+ void DumpBacktrace() {
+-#if V8_LIBC_GLIBC || V8_OS_BSD
+-  void* trace[100];
+-  int size = backtrace(trace, arraysize(trace));
+-  OS::PrintError("\n==== C stack trace ===============================\n\n");
+-  if (size == 0) {
+-    OS::PrintError("(empty)\n");
+-  } else {
+-    for (int i = 1; i < size; ++i) {
+-      OS::PrintError("%2d: ", i);
+-      Dl_info info;
+-      char* demangled = NULL;
+-      if (!dladdr(trace[i], &info) || !info.dli_sname) {
+-        OS::PrintError("%p\n", trace[i]);
+-      } else if ((demangled = abi::__cxa_demangle(info.dli_sname, 0, 0, 0))) {
+-        OS::PrintError("%s\n", demangled);
+-        free(demangled);
+-      } else {
+-        OS::PrintError("%s\n", info.dli_sname);
+-      }
+-    }
+-  }
+-#elif V8_OS_QNX
+-  char out[1024];
+-  bt_accessor_t acc;
+-  bt_memmap_t memmap;
+-  bt_init_accessor(&acc, BT_SELF);
+-  bt_load_memmap(&acc, &memmap);
+-  bt_sprn_memmap(&memmap, out, sizeof(out));
+-  OS::PrintError(out);
+-  bt_addr_t trace[100];
+-  int size = bt_get_backtrace(&acc, trace, arraysize(trace));
+-  OS::PrintError("\n==== C stack trace ===============================\n\n");
+-  if (size == 0) {
+-    OS::PrintError("(empty)\n");
+-  } else {
+-    bt_sprnf_addrs(&memmap, trace, size, const_cast<char*>("%a\n"),
+-                   out, sizeof(out), NULL);
+-    OS::PrintError(out);
+-  }
+-  bt_unload_memmap(&memmap);
+-  bt_release_accessor(&acc);
+-#endif  // V8_LIBC_GLIBC || V8_OS_BSD
```

Comment:
Ah that's what you meant by "comment out *body*".

But wouldn't it be nicer to "#if 0" it?